drinfomon.lic

```diff
@@ -694,9 +721,14 @@ status_action = proc do |server_string|
     DRRoom.room_objs = []
   elsif server_string =~ /^you \w+ (west|east|north|south|northeast|northwest|southeast|southwest|out)\./i
     DRRoom.group_members = []
+  elsif server_string =~ /Obvious exits:/ or server_string =~ /Obvious paths:/
```

Comment:
would you mind changing this to || instead of or? ruby is really weird about or, || is for boolean logic and ‘or’ is for control flow and lower presedence

Reply:
rubocop might catch that actually

Reply:
yea, sarvatts pinned it basically... `or` `and` etc are banned 
```
The and and or keywords are banned. The minimal added readability is just not worth the high probability of introducing subtle bugs. For boolean expressions, always use && and || instead. For flow control, use if and unless; && and || are also acceptable but less clear. [link]
```

Reply:
Found a better method

Reply:
that or basically translates to elsif server_string yadda unless server_string foobar, so easy to have a subtle bug there